frontend/src/App.tsx

```diff
@@ -4,18 +4,34 @@ import useParticipants from "./useParticipants";
 import ParticipantsList from "./ParticipantsList";
 import AddNewParticipant from "./AddNewParticipant";
 import {ToastContainer} from "react-toastify";
+import {HashRouter, Route, Routes} from "react-router-dom";
+import ParticipantDetails from "./ParticipantDetails";
 
 
 export default function App() {
 
     const participantsHook = useParticipants()
 
     return (<>
-            <header>Speed-friending App</header>
-            <h1>Registration</h1>
```

Comment:
Ich bemerke, dass Du das h1-Tag für "Registrierung" verwendet hast, ich bin sicher, dass dies nicht Dein App-Name ist.
In der Web-Dev Welt ist es besser, wenn man nur ein h1-Tag auf der gesamten Website verwendet, das ist besser für SEO.
Ich weiß nicht, ob das in der Softwareentwicklung so auch ist

Reply:
Danke, wusste nicht,  ich werde das ändern!